charts/yurt-app-manager/values.yaml

```diff
@@ -40,10 +41,6 @@ service:
   type: ClusterIP
   port: 443
```

Comment:
maybe the setting of service at here can be removed, because in admissionWebhooks there are service setting definition.

Reply:
fixed